test/test-versions.js

```diff
@@ -106,10 +128,13 @@ function testVersions(dataFilename) {
     }
   }
 
+  /**
+   * @param {Identifier} data
+   */
   function findSupport(data) {
     for (const prop in data) {
-      if (prop === 'support') {
-        checkVersions(data[prop]);
+      if (prop === '__compat' && data[prop].support) {
+        checkVersions(data[prop].support);
```

Comment:
This operation was previously type unsafe, and could lead to a potential crash if there was a feature in BCD named `support`.